src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -75,6 +75,16 @@ class ProjectFileParser {
         }
     }
 
+    File findReferencedDll(String  dllName) {
+        def dllPath = references.find {
+            it.Filename.startsWith(dllName)

```

Comment:
should match the full name (or at least the full name w/o extension) to avoid wrong results (it's very common in C# to have dlls with similar name)

Reply:
The startsWith does not play well with cases where dlls would have a common naming part, like 
Microsoft.Build.dll and Microsoft.Build.Common.dll